docs/INSTALLATION.md

```diff
@@ -1,36 +1,30 @@
 # Installation
+You can download the latest Racket installer from [the official Racket website](https://download.racket-lang.org/).  Installers are available for Windows (64-bit and 32-bit), macOS (Apple Silicon and Intel) and Linux (64-bit).
 
-Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for instructions on downloading and installing Racket.
+If you prefer to use a package manager, like `brew` on MacOS, `winget` on Windows or your Linux distribution's package manager, see [the Racket Wiki](https://github.com/racket/racket/wiki/Installing-Racket) for more detailed instructions.
 
-Alternative installation options and further notes for specific OSes follow.
+Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for guidance materials.
 
-## macOS
+The [Racket Guide](https://docs.racket-lang.org/guide/intro.html) introduces you to interacting with Racket. 
 
-Install via Homebrew:
+See [DrRacket: The Racket Programming Environment](https://docs.racket-lang.org/drracket/index.html) for guidance using the DrRacket IDE.
 
-```
-brew update
-brew install minimal-racket
-```
-The `minimal-racket` package will probably work for solving the problems, but it does not include the graphical interface IDE, DrRacket. If you want that, you have to install the full racket distribution from cask:
+See [Command-Line Tools and Your Editor of Choice](https://docs.racket-lang.org/guide/other-editors.html) for an introduction to racket command line tool and using other editors.
+
+
+## About the Linux installer:
+The Linux build is generic enough that it should work on most distributions, including relatively old distributions. 
 
+After downloading the installer file, run it with
 ```
-brew tap caskroom/cask
-brew install racket
+sh racket-8.8-x86_64-linux-cs.sh
 ```
 
-### macOS official installer
-Note that on macOS, the official installer puts the files under `/Applications`, and as a result the `racket` and `raco` executables are not accessible on command-line terminals. You have to add the `bin` subdirectory below the Racket installation location (e.g., `/Applications/Racket v6.8/bin` for version 6.8) to the `$PATH`.  
+You man need to prefix `sudo` to the start of the command to install to a location that requires adminstrator access.
 
+**Warning: Racket may also be available through your distribution's package manager, although it may be older than the latest Racket version.** While you may choose to do this later, as a learner you will have a much better experience if you stick to the current release available from https://download.racket-lang.org
```

Comment:
This would be great for a special block (see [the docs](https://exercism.org/docs/building/markdown/markdown)):

```suggestion
~~~~exercism/caution
Racket may also be available through your distribution's package manager, although it may be older than the latest Racket version.
While you may choose to do this later, as a learner you will have a much better experience if you stick to the current release available from https://download.racket-lang.org
~~~~
```